BepInEx.Core/BepInEx.Core.csproj

```diff
@@ -7,10 +7,6 @@
     <LangVersion>8</LangVersion>
     <Description>Unity plugin injection framework</Description>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
-    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
```

Comment:
How is this related to semver change? This should be in a separate PR for better testability and ability to revert it if needed.